modules/DMZ/data_kit/dataset_insight.py

```diff
@@ -30,3 +30,33 @@ def get_prediction_type(target_column):
 def get_delimiter(path):
     with open(path, 'rb') as csvfile:
         return csv.Sniffer().sniff(csvfile.read(), delimiters=';,').delimiter
+
+
+# Gets the ratio of missing values to existing values in a dataframe. Either operates on rows or columns, depending
+# on input.
+def get_missing_ratios(pandas_data, method):
+
+    if method == "column":
```

Comment:
Having a toggle for rows vs columns sounds like something we could put in utils. Is there any way we could simplify this to do a toggle and only return a single value? Like,

```
if method == "columns":
    return 0
elif method == "row":
    return 1
```

The **trick** is, you would need to use it like this.

```
axis_toggle = get_toggle(axis)

for i in range(0, pandas_data.shape[axis_toggle]):
    ratio = float(missing_data_counts[i]) / pandas_data.shape[!axis_toggle]
```

I think that's cleaner. What do you think @asclines ?